src/main/java/bio/terra/dao/RelationshipDao.java

```diff
@@ -0,0 +1,51 @@
+package bio.terra.dao;
+
+import bio.terra.configuration.DataRepoJdbcConfiguration;
+import bio.terra.metadata.StudyRelationship;
+import bio.terra.metadata.Study;
+import bio.terra.metadata.StudyTable;
+import bio.terra.metadata.StudyTableColumn;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import java.util.Map;
+import java.util.UUID;
+
+@Repository
+public class RelationshipDao extends MetaDao<StudyRelationship> {
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public RelationshipDao(DataRepoJdbcConfiguration jdbcConfiguration) {
+        jdbcTemplate = new NamedParameterJdbcTemplate(jdbcConfiguration.getDataSource());
+    }
+
+    void createStudyRelationships(Study study) {
+        for (StudyRelationship rel : study.getRelationships().values()) {
+            create(rel, study.getTables());
+        }
+    }
+
+    public void create(StudyRelationship studyRelationship, Map<String, StudyTable> tables) {
```

Comment:
What is the transaction context of the create? I am guessing it assumes it is incorporated into a higher level transaction. Maybe add a comment here with that assumption?